cmd/signal/allrpc/main.go

```diff
@@ -5,8 +5,8 @@ import (
 	"fmt"
 	"os"
 
-	log "github.com/pion/ion-log"
 	"github.com/pion/ion-sfu/cmd/signal/allrpc/server"
+	log "github.com/pion/ion-sfu/pkg/logger"
```

Comment:
An important aspect of this effort is to change ion-sfu's approach of using a global hard-coded logger.  Ultimately, only "main" packages should ever call "github.com/pion/ion-sfu/pkg/logger" and all library code (non-main packages) should be using injected logr.Logger instances.

Is this planned for a follow-up change?